### PR TITLE
Added PR_TEMPLATE

### PR DESCRIPTION
**Pull request by afzal442**
This PR adds template for pull request

### CODE REVIEW

File: .github/PR_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**Notes for Reviewers**
+
+
+**[Signed commits](https://github.com/buildpacks/.github/blob/main/CONTRIBUTING.md#sign-off-process)**
+- [ ] Yes, I signed my commits.
+ 
+
+<!--
+Thank you for contributing to Meshery!
```
Comment (jromero) [inline .github/PR_TEMPLATE.md:21]:
```suggestion
Thank you for contributing to Cloud Native Buildpacks! 
```

File: .github/PR_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ 
+
+<!--
+Thank you for contributing to Meshery! 
+
+Contributing Conventions:
+
+1. Include descriptive PR titles with [<component-name>] prepended.
+2. Build and test your changes before submitting a PR. 
+3. Sign your commits
```
Comment (jromero) [inline .github/PR_TEMPLATE.md:27]:
```suggestion
1. Run `make prepare-for-pr` before submitting a PR. 
2. Sign your commits.
```

**Review by jromero — CHANGES_REQUESTED** [review]

File: .github/PR_TEMPLATE.md (hunk 1 above)
Comment (afzal442) [reply .github/PR_TEMPLATE.md:21]: Oh yes! how can I forget that; 😄

**Review by jromero — APPROVED** [review]
Thanks for the contribution! Keep it up!

**Pull request merged**